airbyte-integrations/connectors/source-harvest/metadata.yaml

```diff
@@ -26,9 +26,14 @@ data:
       enabled: true
     oss:
       enabled: true
+  releases:
+    breakingChanges:
+      1.0.0:
+        message: "This update requires a reset for the following streams to due an update in the format of state: `expenses_clients`, `expenses_categories`, `expenses_projects`, `expenses_team`, `time_clients`, `time_projects`, `time_tasks`, `time_team`, `uninvoiced`, `estimate_messages`, `invoice_payments`, `project_assignments`."
```

Comment:
Please start with "Several changes have been made to the Harvest connector."

Reply:
Updated